blueprint/core/blueprint_View.cpp

```diff
@@ -113,7 +113,7 @@ namespace blueprint
 
     void View::setFloatBounds(juce::Rectangle<float> bounds)
     {
-        static const juce::Identifier transformMatrix("transform-matrix");
+        static const juce::Identifier transformMatrix(transformMatrixProp);
```

Comment:
Should we skip this construction and just call `if(props.contains(transformMatrixProp))` below?

Reply:
Ah what a fool. Silly copy paste/replace. Good spot.